test/test_spectral_ops.py

```diff
@@ -798,14 +797,13 @@ def test_stft_roundtrip_complex_window(self, device, dtype):
             # input
             (torch.randn(600, device=device, dtype=dtype),
              torch.randn(807, device=device, dtype=dtype),
-             torch.randn(12, 14, device=device, dtype=dtype),
-             torch.randn(9, 6, device=device, dtype=dtype)),
+             torch.randn(12, 60, device=device, dtype=dtype)),
```

Comment:
Why change these inputs?

Reply:
reflect mode needs to have enough data to reflect, otherwise you get an error:
```
RuntimeError: Argument #4: Padding size should be less than the corresponding input dimension, but got: [...]
```